src/app/guides/building/page.tsx

```diff
@@ -0,0 +1,11 @@
+import Link from 'next/link';
+
+export default function BuildingRedirect() {
+  return (
+    <>
+      <meta httpEquiv="refresh" content="2;url=/contribute/desktop" />
```

Comment:
Delay 2s to improve accessibility.

Refer to: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/meta#refresh